### ISSUE TITLE
inputs support for mobile device

### ISSUE THREAD

**Issue by liurenqiu520** [jpikl/cfxnes#1]
the inputs just support for pc, hope to support for mobile, thx!

**Comment by jpikl** [comment]
Yes, there is currently no support for that. The app could display on-screen controls when running on a mobile device. 

However, before that, I should do at least some performance optimizations for the emulator to be able to run on mobile devices.
